chapters/annotations.tex

```diff
@@ -227,35 +248,49 @@ \subsubsection{Variable Replacements}\label{variable-replacements}
 \end{center}
 \end{table}
 
+In \lstinline!%{$\mathit{variable}$}!, text markup escape sequences don't apply inside the $\mathit{variable}$, which has the form of \lstinline[language=grammar]!component-reference! in the grammar (\cref{expressions1}).  This means that a complete \lstinline[language=grammar]!component-reference! shall be scanned before looking for the terminating closing brace.
+
+\begin{example}
+The variable replacement \lstinline!%{'%%'}! references the variable \lstinline!'%%'!, not the variable \lstinline!'%'!.
+\end{example}
+
+\begin{example}
+The variable replacement \lstinline!%{foo . '}bar{'}! makes a valid reference to the variable\linebreak[4] \lstinline!foo.'}bar{'!.
+\end{example}
+
 Note that expansion to the final value means that expansion is not restricted to
 parameters and constants, so that values to be shown in a caption can be
 determined during simulation.
 
-The percent character is encoded \lstinline!%%!.  Neither \lstinline!%class! nor
-\lstinline!%name! is supported in this context, as this information is expected
-to already be easily accessible (when applicable) in tool-specific ways. (Titles
-making use of \lstinline!%class! or \lstinline!%name! would then only lead to
-ugly duplication of this information.)
+\begin{nonnormative}
+By design, neither \lstinline!%class! nor \lstinline!%name! is supported in this context, as this information is expected to already be easily accessible (when applicable) in tool-specific ways.  (Titles making use of \lstinline!%class! or \lstinline!%name! would then only lead to ugly duplication of this information.)
```

Comment:
I find ugly a bit too strong and would prefer unnecessary, or some formulation with redundant.

Reply:
Can one say _redundant duplication_, or is _redundant_ redundant in that expression?
> Titles making use of \lstinline!%class! or \lstinline!%name! would then result in redundant duplication of this information.

Reply:
But 'result in redundant information' would work, right?

Reply:
It removes the local redundancy around the word _redundant_, but lacks the explanatory effect of the word _duplication_ with its negative connotation.  Without the explanatory effect, I find the entire sentence redundant in view of the previous sentence.